src/modules/GroupChannel/components/SuggestedReplies/index.tsx

```diff
@@ -11,20 +11,24 @@ export interface SuggestedRepliesProps {
 export interface ReplyItemProps {
   value: string;
   onClickReply: (event: React.MouseEvent<HTMLDivElement>, option: string) => void;
+  type?: 'vertical' | 'horizontal';
 }
 
 export const ReplyItem = ({
   value,
   onClickReply,
+  type = 'vertical',
 }: ReplyItemProps) => {
   return (
-    <div
-      className="sendbird-suggested-replies__option"
-      id={value}
-      onClick={(e) => onClickReply(e, value)}
-    >
-      {value}
-    </div>
+    <>
+      <div
+        className={`sendbird-suggested-replies__option ${type}`}
+        id={value}
+        onClick={(e) => onClickReply(e, value)}
+      >
+        {value}
+      </div>
+    </>
```

Comment:
```suggestion
      <div
        className={`sendbird-suggested-replies__option ${type}`}
        id={value}
        onClick={(e) => onClickReply(e, value)}
      >
        {value}
      </div>
```
It seems we don't need this empty tag here

Reply:
oh weird. I needed it when fixing but now we dont. thanks i removed